tests/controls/listbasetest.cpp

```diff
@@ -224,6 +225,15 @@ void ListBaseTestCase::ItemClick()
     sim.MouseClick(wxMOUSE_BTN_RIGHT);
     wxYield();
 
+    // We want a point within the listctrl but below any items
+    point = list->ClientToScreen(pos.GetPosition()) + wxPoint(10, 50);
+
+    sim.MouseMove(point);
+    wxYield();
+
+    sim.MouseClick();
```

Comment:
This one need to be debugged because a deselection event can be intercepted even without this!

Reply:
With the MSW implementation or generic? I don't see this happening with MSW. For generic there seems to be a FIXME-`#ifndef`, I'll move the `deselected` assert within that and we'll see what happens.